internal/local/structure_v2.go

```diff
@@ -127,21 +109,32 @@ func (a *AppDataV2) unmarshalAuth(rootDir string) error {
 	}
 
 	var auth AuthStructure
-
 	cfg, cfgErr := readFile(filepath.Join(dir, FileConfig.String()))
 	if cfgErr != nil {
 		return cfgErr
 	}
+	if err := unmarshalJSON(cfg, &auth.Config); err != nil {
+		fmt.Println("failed to read config", string(cfg))
```

Comment:
is there a better way to write to stdout here via some CLI util vs. `fmt`? or is this just for debugging?

Reply:
just debug code, thanks for catching